draft-ietf-httpbis-connect-tcp.md

```diff
@@ -156,11 +157,71 @@ Clients SHOULD assume that all proxy resources generated by a single template sh
 
 ## Closing Connections
 
-In each HTTP version, any requirements related to closing connections in Classic HTTP CONNECT also apply to "connect-tcp", with the following modifications:
-
-* In HTTP/1.1, endpoints SHOULD close the connection in an error state to indicate receipt of a TCP connection error (e.g., a TCP RST or timeout).  Acceptable error states include sending an incomplete DATA capsule (as defined in {{Section 3.3 of !RFC9297}}), a TLS Error Alert ({{!RFC8446, Section 6.2}}), or a TCP RST (if TLS is not in use).  When a connection is terminated in an error state, the receiving endpoint SHOULD send a TCP RST if the underlying TCP implementation permits it.
-* In HTTP/2 and HTTP/3, senders MAY use an incomplete DATA capsule to indicate a TCP connection error, instead of (or in addition to) the signals defined for TCP connection errors in Classic HTTP CONNECT.  Recipients MUST recognize any incomplete capsule as a TCP connection error.
-* Intermediaries MUST propagate connection shutdown errors, including when translating between different HTTP versions.
+Connection termination is essentially symmetrical for proxies and their clients.  In this section, we use the term "endpoint" to describe an implementation of this specification in either role.
+
+When closing connections, endpoints are subject to the following requirements:
+
+* When an endpoint receives a valid TCP FIN, it MUST send a FINAL_DATA capsule.
+* When an endpoint receives a valid FINAL_DATA capsule, it MUST send a TCP FIN.
+* Each endpoint MUST close its send stream when the corresponding TCP connection reaches the TIME-WAIT or CLOSED state.
```

Comment:
```suggestion
* When the corresponding TCP connection reaches the TIME-WAIT or CLOSED state, each endpoint MUST close its send stream.
```
Purely editorial, but it might make sense to start with "When" for consistency with the bullet points above.